package.json

```diff
@@ -4,8 +4,10 @@
         "node": ">=16"
     },
     "scripts": {
-        "dev": "vite dev",
-        "build": "node build.js",
+        "dev": "npm-run-all --parallel dev:*",
+        "dev:vite": "vite dev",
+        "dev:i18n": "typesafe-i18n",
+        "build": "node build.js --parallel dev:i18n",
```

Comment:
not sure parallel is gonna work here. It needs to happen before build.js

Reply:
I tried to run prebuild function but it was also not helpful the command is getting stuck and the build is not running.